examples/wafv2-custom-response/main.tf

```
@@ -0,0 +1,90 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "waf" {
+  source = "../.."
+
+  name_prefix          = var.name_prefix
+  allow_default_action = true
+
+  create_alb_association = false
+
+  visibility_config = {
+    cloudwatch_metrics_enabled = false
+    metric_name                = "${var.name_prefix}-waf-setup-waf-main-metrics"
+    sampled_requests_enabled   = false
+  }
+
+  custom_response_bodies = [
+    {
+      key          = "custom_response_body_1",
+      content      = "You are not authorized to access this resource.",
+      content_type = "TEXT_PLAIN"
+    },
+    {
+      key          = "custom_response_body_2",
+      content      = "You there are not authorized to access this resource.",
+      content_type = "TEXT_PLAIN"
+    }
+  ]
+
+  rules = [
+    {
+      name     = "ip-rate-based"
+      priority = "6"
+      action   = "block"
+
+      custom_response = {
+        response_code = 412
+        response_key  = "default_1",
```

```suggestion
        custom_response_body_key  = "default_1",
```
Just a better exposure of the underlying api imho (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-wafv2-webacl-customresponse.html)